effective_posts.gemspec

```diff
@@ -23,4 +23,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'effective_ckeditor'
   s.add_dependency 'effective_datatables', '>= 4.0.0'
   s.add_dependency 'effective_regions'
+  s.add_dependency 'effective_resources'
```

Comment:
We need to include this dependency right?

Reply:
good catch, yes thanks!